proxy/proxy.go

```diff
@@ -51,19 +61,41 @@ func main() {
 
 	loggedRouter := handlers.LoggingHandler(os.Stdout, proxyRouter)
 
-	err = http.ListenAndServe(":"+c.Port, loggedRouter)
+	err = http.ListenAndServe(":" + c.Port, loggedRouter)
 	if err != nil {
 		fmt.Println(err)
 	}
 }
-
+func CallbackUrl() (string, error) {
+	var vcapApp VcapApplication
+	err := json.Unmarshal([]byte(c.VcapAppRaw), &vcapApp)
+	if err != nil {
+		return "", err
+	}
+	// we try first ssl to see if it can be used
```

Comment:
Yes you're right, I think I went too fast. It obviously go to an http request every times. 
Well I will do the change that you just suggested, thank you :)